packages/storybook/cxl-ui/cxl-section-hp-hero.stories.js

```diff
@@ -0,0 +1,49 @@
+import { html } from 'lit';
+import '@conversionxl/cxl-ui/src/components/cxl-section.js';
+import '@vaadin/button';
+import '@vaadin/icon';
+import '@vaadin/horizontal-layout';
+
+const meta = {
+  component: 'cxl-section',
+  title: 'CXL UI/cxl-section-hp-hero',
+};
+
+export default meta;
+
+export const CXLSectionHPHero = {
+  args: {
+    backgroundColor: 'var(--lumo-shade)',
```

Comment:
This token alone kills all text visibility? Dark background, if that's the design, requires a bit more work than this.

Example https://cxl.com/institute/programs/conversion-optimization/

But indeed, this implementation is currently not in CXL UI. Maybe time to bring it?